fastlane/lanes/localization.rb

```diff
@@ -93,6 +93,7 @@
 MANUALLY_MAINTAINED_STRINGS_FILES = {
   File.join('WordPress', 'Resources', 'en.lproj', 'InfoPlist.strings') => 'infoplist.', # For now WordPress and Jetpack share the same InfoPlist.strings
   File.join('WordPress', 'WordPressDraftActionExtension', 'en.lproj', 'InfoPlist.strings') => 'ios-sharesheet.', # CFBundleDisplayName for the "Save as Draft" share action
+  File.join('WordPress', 'JetpackDraftActionExtension', 'en.lproj', 'InfoPlist.strings') => 'ios-jetpack-sharesheet.', # CFBundleDisplayName for the "Save to Jetpack" share action
```

Comment:
By adding the new `InfoPlist.strings` to this hash, out automation will extract its content in the unified `strings` file sent to GlotPress for translation during code freeze.

Then, every time we run a beta and when we'll finalize the release build, it will look for the matching keys in the  GlotPress translations download and extract them in dedicated `WordPress/JetpackDraftAction/<locale>.lproj/InfoPlist.strings` files.

Reply:
NIce!